openxr/src/graphics/headless.rs

```diff
@@ -0,0 +1,60 @@
+use crate::*;
+
+/// Used to create a session without graphics.
+///
+/// See [`XR_MND_headless`] for safety details.
+///
+/// [`XR_MND_headless`]: https://www.khronos.org/registry/OpenXR/specs/1.1/html/xrspec.html#XR_MND_headless
+pub enum Headless {}
+
+impl Graphics for Headless {
+    type Requirements = Requirements;
+    type SessionCreateInfo = SessionCreateInfo;
+    type SwapchainImage = u32;
+    type Format = i64;
```

Comment:
Are these types ever used in a headless session? Might be cleanest to define them as empty enums, if not (`enum HeadlessSwapchainImage {}`, `enum HeadlessFormat {}`).

Reply:
did this and added comments as reminder